Implemented entity Trip

behramip opened the pull request:
->Complete implementation of the Trip Entity
->Creation of empty classes TripDao and TripDaoImpl

behramip committed:
Implemented entity Trip

Complete implementation of the Trip Entity
Creation of empty classes TripDao and TripDaoImpl

behramip committed:
Completed Trip Entity

->Added filePathToPicture parameter
->Implemented hashCode(), equals() and toString() methods

behramip committed:
Implemented TripDaoImpl

behramip committed:
Fixed Trip Entity

MartinMatko commented on TravelAgency-Persistence/src/main/java/cz/muni/fi/pa165/travelAgency/persistence/dao/TripDao.java:
```diff
@@ -0,0 +1,49 @@
+     * find all trips
+     *
+     * @return all trips
+     */
```
I think that findTripByCountry would be nice

MartinMatko commented on TravelAgency-Persistence/src/main/java/cz/muni/fi/pa165/travelAgency/persistence/entity/Trip.java:
```diff
@@ -0,0 +1,194 @@
+    private String filePathToPicture;
+
+    public Trip(){
+        
```
initialise possibleExcursions!!!

behramip committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into patrik

# Conflicts:

#	TravelAgency-Persistence/src/main/java/cz/muni/fi/pa165/travelAgency/persistence/entity/Excursion.java

#	TravelAgency-Persistence/src/main/java/cz/muni/fi/pa165/travelAgency/persistence/entity/Trip.java

behramip committed:
Fix commit

behramip committed:
Fixed Trip constructor

Fixed Trip constructor (initialised Set)
Created empty method findTripsByCountry

TODO: Implement method findTripsByCountry

behramip committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into patrik

behramip committed:
Implemented findByCountry() method

behramip committed:
Created empty ReservationDaoTest.class
